src/main/java/nextstep/fp/Conditional.java

```diff
@@ -0,0 +1,5 @@
+package nextstep.fp;
+
+public interface Conditional {
+    boolean test(Integer number);
+}
```

Comment:
```suggestion
@FunctionalInterface
public interface Conditional {
    boolean test(Integer number);
}
```
@FunctionalInterface를 사용하면 메서드가 정말로 1개만 있는지 컴파일시에 확인해줍니다. 🙂